package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "beacon-for-ibm-dotcom",
+  "name": "@ibmdotcom/beacon-for-ibm-dotcom",
```

Comment:
@kennylam should we just call it `@ibmdotcom/beacon`?

Reply:
Sure, we can. I'll make the change